services/status/locales/i18n.json

```diff
@@ -241,17 +241,29 @@
         "en-tts": "Keyboard layout submenu"
     },
     "mainmenu.battery_disconnect": {
-        "en": "Disconnect battery",
-        "ja": "バッテリーを外します",
-        "zh": "断开电池",
-        "en-tts": "Disconnect battery"
+        "en": "Shutdown",
+        "ja": "シャットダウン",
+        "zh": "关闭",
+        "en-tts": "Shutdown"
+    },
+    "mainmenu.shutdown_confirm": {
+        "en": "Confirm shutdown:\nDevice will reamin off until a charging cable is plugged in. Proceed?",
+        "ja": "シャットダウンの確認:\n充電ケーブルが差し込まれるまで、デバイスはオフのままになります。続行しますか?",
+        "zh": "确认关机：\n设备将保持关闭状态，直到插入充电线。继续？",
+        "en-tts": "Confirm shutdown:\nDevice will reamin off until a charging cable is plugged in. Proceed?"
     },
     "mainmenu.reboot": {

```

Comment:
This language entry is not needed anymore

Reply:
Actually, an extra screen was added if you select "shutdown" just to make sure that the user is aware that selecting this would require you to have a charging cable handy to bring the device back. If you accidentally select it while you're out and about and don't have a charging cable it would be nice to be able to "nope out" of the process, I think...